apps/ca.c

```diff
@@ -209,7 +209,7 @@ const OPTIONS ca_options[] = {
     {"noemailDN", OPT_NOEMAILDN, '-', "Don't add the EMAIL field to the DN"},
 
     OPT_SECTION("Signing"),
-    {"md", OPT_MD, 's', "md to use; one of md2, md5, sha or sha1"},
+    {"md", OPT_MD, 's', "md to use; (e.g., sha256)"},
```

Comment:
Did you mean to add the ','

Reply:
> Did you mean to add the ',' after e.g.?

Yes.  See https://www.grammarly.com/blog/know-your-latin-i-e-vs-e-g/ for  example.

Reply:
The examples use both forms.. Thanks for the info.

Reply:
I think this is a UK vs US difference.

Although the reason I understand for including the comma is to mark the beginning of a list and this is no longer a list.

Reply:
Rewrote the text to avoid the issue.